docs/index.md

```diff
@@ -124,11 +126,11 @@ Lieer can be configured using `gmi set`. Use without any options to get a list o
 **`Replace slash with dot`** is used to replace the sub-label separator (`/`) with a dot (`.`). I think this is easier to work with. *Important*: See note below on [changing this setting after initial sync](#changing-ignored-tags-and-translation-after-initial-sync).
 
 **`Ignore tags (local)`** can be used to specify a list of tags which should not be synced from local to remote (e.g. [`new`](#usage)). In addition to the user-configured tags these tags are ignored: `'attachment', 'encrypted', 'signed', 'passed', 'replied', 'muted', 'mute', 'todo', 'Trash', 'voicemail'`. Some are special tags in notmuch and some are unsupported by GMail. See [Caveats](#caveats) below for more explanations. *Note:* This setting expects [_translated_ tags](#translation-between-labels-and-tags).
-  
+
```

Comment:
These whitespaces are there to create new paragraphs and should be left in.

Reply:
Sorry, these whitespaces are automatically deleted by vim when saving the file. But don't we only need newline to create a new paragraph?

Reply:
No, I think they are needed. But if it can be solved with an extra newline  then you could try that. Should be observable on your branch on github.